### PR TITLE
Order completed and Contact Us pages

### PR DESCRIPTION
**Pull request by cami65**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -45,22 +45,22 @@ export const Header = () => {
...
-              <a className="nav-link" href="App.tsx">
+              <a className="nav-link" href="/">
                 Blogs
               </a>
-              <a className="nav-link" href="App.tsx">
+              <a className="nav-link" href="/shopping-cart">
                 Shop
               </a>
-              <a className="nav-link" href="App.tsx">
+              <a className="nav-link" href="/contact-us">
```
Comment (Negodyaj) [inline src/components/Header/Header.tsx]: you must change these a-tags into Link components

File: src/pages/ContactUs/ContactUs.tsx [hunk 2]
```diff
@@ -1,5 +1,92 @@
 import './ContactUs.scss';
+import hehehe from './photo/Group 124.svg';
```
Comment (Negodyaj) [inline src/pages/ContactUs/ContactUs.tsx:2]: hehehe?) really?)

File: src/pages/ContactUs/ContactUs.tsx [hunk 3]
```diff
@@ -1,5 +1,92 @@
...
+                  <div >
+                    <input className='subject' typeof="text" placeholder="Subject*"></input>
+                  </div>
+                  <div>
+                    <input className='message-one' type=" text" placeholder='Type Your Message*' />
+                  </div>
+                </div>
+                <div className="button">
+                  <button className="send">
+                    <p>Send Mail</p>
```
Comment (Negodyaj) [inline src/pages/ContactUs/ContactUs.tsx]: you don't need to create a paragraph for button's content

File: src/pages/ContactUs/ContactUs.tsx (hunk 2 above)
Comment (cami65) [reply src/pages/ContactUs/ContactUs.tsx:2]: I couldn't think of a suitable name at that moment)))))))))

File: src/pages/ContactUs/ContactUs.tsx (hunk 3 above)
Comment (cami65) [reply src/pages/ContactUs/ContactUs.tsx]: Ok) i will correct  it

Comment (Negodyaj) [reply src/pages/ContactUs/ContactUs.tsx]: any progress?)

**Pull request merged**